### ISSUE TITLE
Why are OTSi attributes in a separate transponder-list sub-tree?

### ISSUE THREAD

**Issue by dieterbeller** [ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang#5]
It is unclear why the transponder attributes have been put into a separated sub-tree (see snippet from the current tree-view below) and have not be added to the list of OTSi attributes. What is the rationale?

```
  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
    +--ro OTSiG-element* [OTSiG-identifier]
    |  +--ro OTSiG-identifier    int16
    |  +--ro OTSiG-container
    |     +--ro OTSi* [OTSi-carrier-id]
    |        +--ro OTSi-carrier-id           int16
    |        +--ro OTSi-carrier-frequency?   decimal64
    |        +--ro OTSi-signal-width?        decimal64
    |        +--ro channel-delta-power?      decimal64
    +--ro transponders-list* [transponder-id]
       +--ro transponder-id                   uint32
       +--ro (mode)?
       |  +--:(G.692.2)
       |  |  +--ro standard_mode?             layer0-types:standard-mode
       |  +--:(organizational_mode)
       |  |  +--ro operational-mode?          layer0-types:operational-mode
       |  |  +--ro organization-identifier?   layer0-types:vendor-identifier
       |  +--:(explicit_mode)
       |     +--ro available-modulation*      identityref
       |     +--ro modulation-type?           identityref
       |     +--ro available-baud-rates*      uint32
       |     +--ro configured-baud-rate?      uint32
       |     +--ro available-FEC*             identityref
       |     +--ro FEC-type?                  identityref
       |     +--ro FEC-code-rate?             decimal64
       |     +--ro FEC-threshold?             decimal64
       +--ro power?                           int32
       +--ro power-min?                       int32
       +--ro power-max? int32
```

**Comment by sergiobelotti** [comment]
> It is unclear why the transponder attributes have been put into a separated sub-tree (see snippet from the current tree-view below) and have not be added to the list of OTSi attributes. What is the rationale?
> 
> ```
>   augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
>     +--ro OTSiG-element* [OTSiG-identifier]
>     |  +--ro OTSiG-identifier    int16
>     |  +--ro OTSiG-container
>     |     +--ro OTSi* [OTSi-carrier-id]
>     |        +--ro OTSi-carrier-id           int16
>     |        +--ro OTSi-carrier-frequency?   decimal64
>     |        +--ro OTSi-signal-width?        decimal64
>     |        +--ro channel-delta-power?      decimal64
>     +--ro transponders-list* [transponder-id]
>        +--ro transponder-id                   uint32
>        +--ro (mode)?
>        |  +--:(G.692.2)
>        |  |  +--ro standard_mode?             layer0-types:standard-mode
>        |  +--:(organizational_mode)
>        |  |  +--ro operational-mode?          layer0-types:operational-mode
>        |  |  +--ro organization-identifier?   layer0-types:vendor-identifier
>        |  +--:(explicit_mode)
>        |     +--ro available-modulation*      identityref
>        |     +--ro modulation-type?           identityref
>        |     +--ro available-baud-rates*      uint32
>        |     +--ro configured-baud-rate?      uint32
>        |     +--ro available-FEC*             identityref
>        |     +--ro FEC-type?                  identityref
>        |     +--ro FEC-code-rate?             decimal64
>        |     +--ro FEC-threshold?             decimal64
>        +--ro power?                           int32
>        +--ro power-min?                       int32
>        +--ro power-max? int32
> ```

Please see comment https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/issues/12#issuecomment-507734476
This is one possible answer to why OTSi and Transponder list can (or should ) be separated list

**Comment by sergiobelotti** [comment]
This issue is related to transponder model and it is solved in PR#36
